app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,12 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link rel="icon" type="image/x-icon" href="https://assets-cdn.github.com/favicon.ico">
   </head>
 
   <body>
+    <%= render 'top' %>
```

Comment:
Please use here ```render partial```.

Reply:
Also note that explicitly specifying :partial is required when passing additional options such as :layout.
This is from the documentation on Rails. Does not work.